src/feature.js

```diff
@@ -413,24 +413,37 @@ var feature = function (arg) {
   ////////////////////////////////////////////////////////////////////////////
   /**
    * Get/Set visibility of the feature
+   *
+   * @param {boolean|undefined} val: undefined to return the visibility, a
+   *    boolean to change the visibility.
+   * @param {boolean} direct: if true, when getting the visibility, disregard
```

Comment:
`direct` is only ever passed internally -- the user should never have to reference it.  It is only so we know what the base visibility of a feature is regardless of the layer.